examples/gradle-examples/README.md

```diff
@@ -0,0 +1,23 @@
+# kubernetes-java-annotations Example
+####Kubernetes java annotations Example with Spring Boot and Gradle
+
+#####Step 1:
+run gradlew build
+
+#####Step 2:
+Check the generated Kubernetes deployment scripts under:
+./build/classes/java/main/META-INF/ap4k
+
+E.g:./build/classes/java/main/META-INF/ap4k/kubernetes.yml
+
+#####Step 3:
+Run the kubernetes.yml into Kubernetes, or use the script:
+deployInKubernetes.bat, 
+Or just run the Kubectl command:
+kubectl apply -f ./build/classes/java/main/META-INF/ap4k/kubernetes.yml
+
+Thank you !
+#####Hi, my name is Alexandru Lungu and I will like to send some thanks to Iocanel, the author of the kubernetes annotations plugin.
```

Comment:
As we are going to merge this example directly in ap4k examples, we should remove this reference.

You are welcome BTW!